assets/js/modules/pagespeed-insights/components/common/Recommendation.js

```diff
@@ -38,6 +39,10 @@ export default function Recommendation( props ) {
 		strategy,
 	} = props;
 
+	const onOpen = useCallback( () => {
+		trackEvent( 'pagespeed_widget', 'stack_pack_expand', auditID );
```

Comment:
@felixarntz maybe we should limit this event tracking to only one per stack pack per page view? In other words, if a user opens and closes the `Remove unused CSS` accordion item three times during a single page view, we will track the appropriate event only once instead of three. What do you think?

Reply:
I think we should keep this as is for now and track every user interaction.